Misc/NEWS.d/next/Library/2020-04-14-16-18-49.bpo-40270.XVJzeG.rst

```diff
@@ -0,0 +1,2 @@
+The in-built sqlite extension on Windows is now compiled with the json
```

Comment:
I'd phrase this more like "The included copy of sqlite3 on Windows ..."; it's not the extension that's now compiled that way, it's sqlite3 itself.